usaspending_api/common/mixins.py

```diff
@@ -136,7 +136,11 @@ def filter_records(self, request, *args, **kwargs):
         if len(request.data):
             fg = FilterGenerator()
             filters = fg.create_from_request_body(request.data)
-            return queryset.filter(filters).distinct()
+            try:
+                filtered_qs = queryset.filter(filters).distinct()
+            except:
```

Comment:
AFAIK, a bare "except" with a bare "raise" is effectively the same as not having a try block at all, so this seems a little confusing.